thesis-de/content-de.tex

```diff
@@ -327,6 +327,28 @@ \subsection{Datenerfassungssysteme}
 \subsection{Datenbanksystem / API}
 
 \subsection{Frontend}
+Die Weboberfläche besitzt zwei Schnittstellen, welche aufgeteilt sind, mit der
+Abfrage der Daten an dem Datenhaltungssystem und zum anderen mit der Ausgabe
+und Anzeige für den User bzw. dem Administrator. Eine Schnittstelle auf einem
+Webinterface dient in erster Linie zur Anzeige von Daten, die über ein anderes
+System(BackEnd) erhalten wurden. 
+
+Die erste Schnittstelle der Weboberfläche hat nur eine lesende Berechtigung,
+auf den Datenhaltungssystemen, da keine Änderung durch die Weboberfläche
+durchgeführt werden muss. Die Schnittstelle zwischen den Datenhaltungssystemen
+und der Weboberfläche kann somit mit einer sogenannten
+API~\ref{subsec:schnittstelle_datenbereitstellung} realisiert werden. Somit
+kann hier keine Änderung durch dritter an den Datenhaltungssystemen
+durchgeführt werden.
+
+Die zweite Schnittstelle ist die Ausgabe der durch die Datenhaltungssystem
+erhaltenen Daten in einem Graphen~\ref{definition_eines_graphen} auf der
+Weboberfläche. Die Übertragung der Ausgabe der Daten, bzw. das Aufrufen der
+Weboberfläche wird über ein gesichertes Übertragungsprotokoll mit HTTPS
+realisiert. Dies dient zum einem für die Verschlüsselung der Verbindung von
```

Comment:
Hier mehr Text zu: Das das geil ist, weil der Mensch nun vorher unstrukturierte Daten bla bla in einem Graphen/Weboberfläche/Anzeige auf einem Blick, schnell und vollstänig bla bla sehen und lesen und interpretieren bla bla kann und zum schluss bei bedarf auch exportieren und das Abbild festhalten kann bla bla